.changeset/purple-monkeys-refuse.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/date-picker': patch
+'@twilio-paste/core': patch
```

Comment:
@TheSisb should these be minor/major changes? What about for Input and Input Box (the only change was adding type 'date' to both) - patch?